### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by abeyang00** [FangGet/tf-monodepth2#3]
![Screenshot from 2019-09-25 15-41-14](https://user-images.githubusercontent.com/36392866/65575498-f0dd9a80-dfaa-11e9-8cf4-98fcca172ab2.png)

After training for about 4 epochs, i get results like above. Is this the issue with the input data? or something else?

**Comment by FangGet** [comment]
Sorry for late reply, I upload a modified version, fix small bugs, just try this.
For your question, I found it's due to start learning rate and weight initializer, for now, I decrease start learning rate to 1e-5 for first 5000 iterations and increase it back to 1e-4.
Just trying to fix this problem.
